src/components/SearchForm/SearchForm.js

```diff
@@ -15,66 +15,150 @@ class SearchForm extends Component {
     super(props)
     
     this.state = {
-      formLayout: 'vertical'
+      formLayout: 'vertical',
+      maxTime: '',
+      haveIngredients: [],
+      dontHaveIngredients: [],
+      allergies: [],
+      specificDiets: [],
+      likeCuisines: [],
+      dontLikeCuisines: [],
+      wantCourses: [],
+      dontWantCourses: [],
+      wantOnHolidays: [],
+      dontWantOnHolidays: []
     }
   }
 
-  render(){
-
-    const { formLayout }   = this.state.formLayout;
+  onSubmit(e) {
```

Comment:
will you be able to create a form submission to this api? 
as told you before - don't have exact name for method that will apply for all filters, so just include backend urls with .env file as we did on your repository.

Btw, netfily deploy tells that this commit generate this issue
`4:08:00 PM: Build exceeded maximum allowed runtime
4:08:03 PM: Failed to compile.
4:08:03 PM: ./node_modules/rc-editor-core/node_modules/lodash/_coreJsData.js
4:08:03 PM: Module build failed: Error: ENOENT: no such file or directory, open '/opt/build/repo/node_modules/rc-editor-core/node_modules/lodash/_coreJsData.js'
4:08:03 PM: Compiling...
4:08:04 PM: Failed to compile.
4:08:04 PM: ./src/index.js
4:08:04 PM: Module build failed: Error: ENOENT: no such file or directory, open '/opt/build/repo/src/index.js'`

Reply:
I checked the log and found that the command there is npm start instead of npm run build so you will have to change that

Reply:
got it!

Reply:
ok, i'll merge PR and then will try a deploy again